js/build_map.js

```diff
@@ -6,10 +6,10 @@ var chart_height    =   700;
 var tempCategories = ["total", "thermoelectric", "publicsupply", "irrigation", "industrial"];
 
 // Projection
-var projection = albersUsaTerritories()
-    .scale([1200])
+projection.scale([1200])
```

Comment:
you should be able to make just one projection.scale() call, too - i think you can take out this call here, only define the projection and projection details within map_utils.js. but could you do me a favor and copy over the comment ` // default is .rotate([96,0]) to center on US (we want this)` to map_utils.js? took me hours to figure that out =)